modules/libarchive/3.7.3/MODULE.bazel

```diff
@@ -0,0 +1,15 @@
+module(
+    name = "libarchive",
+    version = "3.7.3",
```

Comment:
would you mind doing this on 3.7.4? That's the latest version in the BCR now

Reply:
I was thinking of doing `3.7.4` as a follow up once we felt this patch was good, but how about I just include both `3.7.3` and `3.7.4.bcr.1` in this PR? :)

Reply:
whatever's easier for you :)

Reply:
once all the other conversations are resolved I'll add `3.7.4.bcr.1` :+1: